api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -77,26 +80,24 @@
 
     @ParameterizedTest
     @MethodSource("allEligibilityOutcomeValues")
-    void shouldReturnDuplicateWhenLiveClaimAlreadyExists(EligibilityOutcome eligibilityOverrideOutcome) {
+    void shouldReturnDuplicateWhenLiveClaimAlreadyExists(EligibilityOverride eligibilityOverride) {
         //Given
         UUID existingClaimId = UUID.randomUUID();
         given(claimRepository.findLiveClaimWithNino(any())).willReturn(Optional.of(existingClaimId));
 
         //When
-        EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementService.evaluateNewClaimant(CLAIMANT, eligibilityOverrideOutcome);
+        EligibilityAndEntitlementDecision decision = eligibilityAndEntitlementService.evaluateNewClaimant(CLAIMANT, eligibilityOverride);
 
         //Then
         assertThat(decision.getEligibilityStatus()).isEqualTo(EligibilityStatus.DUPLICATE);
         assertThat(decision.getExistingClaimId()).isEqualTo(existingClaimId);
         verifyNoInteractions(client);
     }
 
-    private static Stream<EligibilityOutcome> allEligibilityOutcomeValues() {
+    private static Stream<EligibilityOverride> allEligibilityOutcomeValues() {
         return Stream.of(
                 null,
-                EligibilityOutcome.CONFIRMED,
-                EligibilityOutcome.NOT_CONFIRMED,
-                EligibilityOutcome.NOT_SET
+                eligibilityConfirmedWithNoChildrenForFiveYears()
```

Comment:
Shouldn't we add in values for the other EligibilityOutcomes within EligibilityOverride objects, seems like we've deleted some tests here by doing this.

Reply:
I think we need to rename the method. We're actually only testing that a duplicate outcome is returned with or without an eligibilityOverride